fmlcore/src/main/java/net/minecraftforge/fml/OptionalMod.java

```diff
@@ -203,7 +203,7 @@ public T orElseGet(Supplier<? extends T> other) {
      * Return the contained mod object, if present, otherwise throw an exception
      * to be created by the provided supplier.
      *
-     * @apiNote A method reference to the exception constructor with an empty
+     * API Note: A method reference to the exception constructor with an empty
```

Comment:
What's the point in doing this if you added support for the annotation? Also, if continuing to do this, should add `<p>`.

Reply:
This change was done before I re-added support for those JDK-specific annoatations (after I realized that those were used a lot more often than I thought). I'll be reverting that.